.env.example

```diff
@@ -60,8 +60,7 @@ DB_LIST_FOREIGN_KEYS=true
 # LYCHEE_DIST_URL="dist/"
 # LYCHEE_SYM_URL="sym/"
 
-# Support for token based authentication like basic auth. Enabled by default.
-# Disable when running Lychee behind 3rd party managed authentication.
+# Support for token based authentication used by API requests. Enabled by default.
 # ENABLE_TOKEN_AUTH="true"
```

Comment:
Question for the room: For consistency, should we drop the quotes?
```suggestion
# ENABLE_TOKEN_AUTH=true
```

Reply:
You're right. That was a leftover from being a string value at first.